btree/btree.c

```diff
@@ -7698,7 +7686,7 @@ static void ut_basic_tree_oper_cp(void)
 	rc = M0_BTREE_OP_SYNC_WITH_RC(&b_op, m0_btree_destroy(b_op.bo_arbor,
 							      &b_op, tx));
 	M0_ASSERT(rc == 0);
-	m0_free(btree);
+	M0_SET0(&btree);
```

Comment:
This call should be moved above to line # 7682. Please make similar change to call the SET0 after every close.

Reply:
Updated.